index.html

```diff
@@ -34,5 +39,98 @@ <h2>Introduction</h2>
         for how toget started!
       </p>
     </section>
+
+    <section class="informative">
+      <h2>
+        Incubation Usage Examples
+      </h2>
+      <p>
+        The following shows a [=manifest=] that prefers the <code>minimal-ui
+        </code> <a data-cite="appmanifest#dfn-display-mode">display modes</a>
+        to <code>standalone</code>, but if minimal-ui isn't supported, fall
+        back to standalone as opposed to browser.
+      </p>
+      <pre class="example json" title="Advanced display usage manifest">
+        {
+          "name": "Recipe Zone",
+          "description": "All of the recipes!",
+          "icons": [{
+            "src": "icon/hd_hi",
+            "sizes": "128x128"
+          }],
+          "start_url": "/index.html",
+          "display_override": ["minimal-ui"],
+          "display": "standalone",
+          "theme_color": "yellow",
+          "background_color": "red"
+        }
+      </pre>
+    </section>
+
+    <section>
+      <h2 data-dft-for="">Display_override extension</h2>
+      <p>
+        For advanced usages, the display_override member can be used to specify
+        a custom fallback order of {DisplayModeType}}s for developers to choose
+        their prefered <a data-cite="appmanifest#dfn-display-mode">display mode
+        </a>.
+      </p>
+      <p>
+        The <dfn>extension point for evaluating display_override</dfn> is given
+        by the following algorithm addition to the
+        <a data-cite="appmanifest#dfn-steps-for-determing-the-web-app-s-chosen-display-mode">
+        steps for determining the web app's chosen display mode</a>. The
+        following algorithm is executed at the
+        <a data-cite="appmanifest#dfn-extension-point">extension point</a>:
```

Comment:
I find this paragraph a little hard to read.

I think this should use the text in [Web Share Target](https://w3c.github.io/web-share-target/level-2/#extension-to-the-web-app-manifest):

"The following steps are added to the `<a>`extension point`</a>` in `<a>`determining the web app's chosen display mode`</a>`:"